### PR TITLE
Dump stacktrace for crashed instance started by go test in CI

### LINKED ISSUE #1052: Upload go case log if integration tests failed
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.

### Motivation

Otherwise, it's hard to debug and get the root cause of failure in CI environment.

### Solution

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by PragmaTwice**
It closes #1052.

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]
Please update the PR description.

**Comment by PragmaTwice** [comment]
> Please update the PR description.

Done. Thanks.

File: .github/workflows/kvrocks.yaml [hunk 1]
```diff
@@ -217,15 +225,35 @@ jobs:
...
           fi
 
+      - uses: actions/upload-artifact@v2
+        if: ${{ failure() && startsWith(matrix.os, 'ubuntu') }}
+        with:
+          name: kvrocks-coredumps-${{ matrix.name }}
+          path: |
+            ./build/kvrocks
+            ./coredumps/*
+
```
Comment (tisonkun) [inline .github/workflows/kvrocks.yaml:256]:
```suggestion
            ./coredumps/*
            ./tests/gocase/workspace/*
```
?

Comment (PragmaTwice) [reply .github/workflows/kvrocks.yaml:256]: I think the workspace is too large (~n GB), maybe we need to select some specific files.

Comment (tisonkun) [reply .github/workflows/kvrocks.yaml:256]: OK. Then let's run with current solution and add more if necessary.

**Comment by tisonkun** [comment]
Merging...

**Pull request merged**